x-pack/plugins/aiops/public/components/spike_analysis_table/spike_analysis_table.tsx

```diff
@@ -169,13 +170,16 @@ export const SpikeAnalysisTable: FC<SpikeAnalysisTableProps> = ({
           </>
         </EuiToolTip>
       ),
-      render: (_, { histogram, fieldName, fieldValue }) => (
-        <MiniHistogram
-          chartData={histogram}
-          isLoading={loading && histogram === undefined}
-          label={`${fieldName}:${fieldValue}`}
-        />
-      ),
+      render: (_, { histogram, fieldName, fieldValue }) => {
+        if (!histogram) return NOT_AVAILABLE;
```

Comment:
Is this change necessary? The `chartData` prop for the `MiniHistogram` component is optional. Returning like this will also not trigger `isLoading` for the mini histogram component.